src/components/spaces/show/header.js

```diff
@@ -65,11 +105,24 @@ export class SpaceHeader extends Component {
                 {ownerName}
               </a>
             }
-            {editableByMe &&
+
+            {isPrivate && editableByMe &&
+              <ShareableLinkOption
+                shareableLinkUrl={shareableLinkUrl}
+                onEnable={onEnableShareableLink}
+                onDisable={onDisableShareableLink}
+                onRotate={onRotateShareableLink}
+              >
+                <div className='ui image label' >
+                  <Icon name='link'/> Link
```

Comment:
Link is probably a bit confusing. Maybe 'Shareable Link'? But your call either way.